test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -76,14 +76,12 @@ def test_execjob_end_non_blocking(self):
         j.set_sleep_time(1)
         self.server.create_import_hook(hook_name, attr, hook_body)
         jid = self.server.submit(j)
-        (_, str1) = self.mom.log_match(
-                                       "Job;%s;executed execjob_end hook" %
+        (_, str1) = self.mom.log_match("Job;%s;executed execjob_end hook" %
                                        jid, n=100, max_attempts=10, interval=2)
         date_time = str1.split(";")[0]
         epoch = int(time.mktime(time.strptime(
             date_time1, '%m/%d/%Y %H:%M:%S')))
-        (_, str1) = self.mom.log_match(
-                                       "executed exechost_periodic hook",
+        (_, str1) = self.mom.log_match("executed exechost_periodic hook",
```

Comment:
@saimanikant - let us display the log times to be sure so that the sequence is evident in the test output.

Reply:
@prakashcv13 displaying the time is good but how is it going to help if those timings are not right. test should have a check in itself that date_time2 < date_time3. or use starttime and endtime attributes of log_match to make sure periodic hook related messages appears between them.

Reply:
Ah yes missed to see that. Either update the interval=1 to make sure this message appears within 10s or add an end time.

Reply:
consider using convert_date_time() from logutils here and in other places in the test